pkg/session_recording/recordings.jsx

```diff
@@ -529,7 +531,13 @@
         }
 
         componentDidMount() {
-            this.journalctlStart();
+            let proc = cockpit.spawn(["getent", "passwd", "tlog"]);
+
+            proc.stream((data) => {
+                this.uid = data.split(":",3)[2];
+                this.journalctlStart();
```

Comment:
Maybe, I'm missing something here, but it has only worked with .stream. I'll check this once again, maybe it should be called somehow differently.